themes/opentermsarchive/layouts/partials/head.html

```diff
@@ -17,4 +17,14 @@
   <link rel="mask-icon" href="/favicon/safari-pinned-tab.svg" color="#000">
   <meta name="msapplication-TileColor" content="#ffffff">
   <meta name="theme-color" content="#ffffff">
+  <meta property="og:title" content="{{ with .Param "html_title" }}{{ . }}{{ else }}{{ .Title }}{{ end }}">
+  <meta property="og:type" content="{{ if .IsPage }}article{{ else }}website{{ end }}">
+  <meta property="og:url" content="{{ .Permalink }}">
+  {{ $openGraphImage := resources.Get "/images/opengraph/open-terms-archive.jpg" }}
+  {{ with resources.Get (printf "/images/opengraph/%s/%s.jpg" .Page.Lang (.Title | urlize)) }}
+    {{ $openGraphImage = . }}
+  {{ end }}
+  <meta property="og:image" content="{{ $openGraphImage.Permalink }}">
+  <meta property="og:image:width" content="1200">
+  <meta property="og:image:height" content="630">
```

Comment:
Why not use the exact image width and height? 🙂
https://gohugo.io/content-management/image-processing/#image-rendering